config.yml

```diff
@@ -1,22 +1,30 @@
+# Configuration for Rasa NLU.
```

Comment:
we could also use the auto config feature?

Reply:
I did, but it added ConveRT, since we're still on RC2 here. If I left everything commented, then `rasa train` erased my changes and applied the defaults again. I suggest we use the autoconfig on RC3 onwards (this is somewhat of an exceptional situation).